at-tanvo/Exercise1.rb

```diff
@@ -0,0 +1,76 @@
+class News
+  attr_accessor :id, :title, :content
+  def self.update(news, title, content)
+    news.title = title
+    news.content = content
+  end
+  def self.delete(news)
+    @arr.delete(news)
+  end
+  
+end
+class BbcNews < News
+  attr_accessor :category
+  @arr = []
+  def initialize(id, title, content, category)
+    @id = id
+    @title = title
+    @content = content
+    @category = category
+  end
+  def self.news(news)
+    @arr.push(news)
+  end
+  def self.show
+    arr_show = []
+    @arr.each do |x|
+      hash = {}
+      hash[x.id] = [x.title, x.content, x.category]
+      # title = x.title
+      # content = x.content
+      # category = x.category
+      # hash[title] = x.title
+      # hash[content] = x.content
+      # hash[category] = x.category
+      arr_show.push(hash)
+    end
+    arr_show  
+  end  
+end
+class CnnNews
+  attr_accessor :publication_date
+  @arr = []
+  def initialize(id, title, content, publication_date)
+    @id = id
+    @title = title
+    @content = content
+    @publication_date = publication_date
+  end
+  def self.news(news)
+    @arr.push(news)
+  end  
+  def self.show
+    arr_show = []
+    @arr.each do |x|
+      hash = {}
+      title = x.title
+      content = x.content
+      publication_date = x.publication_date
+      hash[title] = x.title
+      hash[content] = x.content
+      hash[publication_date] = x.publication_date
+      arr_show.push(hash)
+    end
+    arr_show
+  end  
+end
+class User
+  attr_accessor :id, :name, :friends, :messages
+end
+bbc_new_01 = BbcNews.new(01,'Con ngu','con du','con cho')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2df379029baf7c4dca775a9bdeb6c1da625ce3d5/at-tanvo/Exercise1.rb#L70"></span>Style/NumericLiteralPrefix: Use 0o for octal literals. (https://github.com/rubocop-hq/ruby-style-guide#numeric-literal-prefixes)</td>
    </tr>
  </tbody>
</table>